README.md

```diff
@@ -3,19 +3,21 @@
 Anonymous DAO is a decentralized alliance of individuals, government, businesses, and nonprofits devoted to the
 minimization of suffering.
 
-To achieve this, we propose the creation of an open-source decentralized health platform
-that will produce a massive acceleration of large-scale research to discover new ways to prevent and treat chronic diseases.
+To achieve this, we propose the creation of an open source software framework for aggregating, managing, processing and storing of health data for better interoperability, portability, availability, analysis, interpretation and security.
 
-It will overcome the traditional barriers to collaboration and data sharing by encoding all data and
-intellectual property contributions to the platform in a non-fungible token (NFT). Using smart contracts, the platform will be able to compensate all contributors with royalties for their contributions.
+With that basic technology layer in place it will help health application businesses and researchers to produce a massive acceleration of large-scale research to discover new ways to prevent and treat chronic diseases, to create precision medicine and ultimately to new insights for more health performance and a longer health span.
+
+It will overcome the traditional barriers to collaboration and data sharing by encoding all data and intellectual property contributions to the platform in a non-fungible token (NFT). Using smart contracts, the platform will be able to compensate all contributors with royalties for their contributions.
 
 # Join Us Right Now!
 
 Hey, you! 👀
 
-Anonymous DAO is open to anyone interested in alleviating the suffering of chronic diseases. Our success in
+Anonymous DAO is open to anyone interested in alleviating the suffering from chronic diseases and from dying of degenerative diseases. Our success in
```

Comment:
How about, "Anonymous DAO is open to anyone interested in preventing suffering and death from chronic diseases and degenerative diseases."? @vincenzodomina